### PR TITLE
Capstone project Pull request 

### PR DESCRIPTION
**Pull request by youmari**
# Details about the project
 
## Mobile version :

- Creating Home page and About page
- Make interaction buttons and links like mobile menu and links
- Make the website responsive for mobile menu
- Using the HTML semantics tags rather than (div)
## You can check mobile-version pull request for documented commits
- Make the buttons and links functioning
- Add Close button for the menu  

# Desktop version : 

- Font size to all the texts
- Add the account bar above the navbar
- Add navbar for desktop version and hide the mobile menu on bigger screens
- Add links inside the navbar
- Make grid for speaker section for responsiveness
- Modify the program section as the same as the desktop version

# Dynamic speakers card

- Create elements for speaker card with image and name and about the speaker
- Generate the speaker's cards dynamically by storing data in objects
- Fix speaker cards styling and eslint error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,47 @@
...
-- Frameworks
-- Technologies used
+- HTMl5
+- CSS3
+- Vanilla JavaScript
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo link ](youmari.github.io/capstone-project1/)
```
Comment (GzimAsani) [inline README.md]: - [x] Live link is return 404 error please update it

Comment (GzimAsani) [reply README.md]: - [x] Please record a video presenting the features of the project.

File: index.html [hunk 2]
```diff
@@ -6,16 +6,29 @@
...
-        <li class="menu-list"><a class="menu-link" href="./html/about.html">About</a</li>
+        <li class="menu-list homepage"><a class="menu-link" href="./index.html">Home Page</a></li>
+        <li class="menu-list"><a class="menu-link" href="./html/about.html">About</a></li>
+        <li class="menu-list"><a class="menu-link" href="#program">Program</a></li>
+        <li class="menu-list"><a class="menu-link" href="./html/about.html">Join</a></li>
+        <li class="menu-list"><a class="menu-link" href="#">Sponsor</a></li>
+        <li class="menu-list"><a class="menu-link" href="./html/about.html">News</a></li>
       </ul>
     </nav>
   </header>
```
Comment (GzimAsani) [inline index.html:34]: - [x] For desktop version please adjust colors and the size of the navbar according to the given design:
Yours:
Screenshot from 2021-11-16 17-42-21
Given Design:
Screenshot from 2021-11-16 17-51-31

File: index.html [hunk 3]
```diff
@@ -29,7 +42,7 @@ <h2 class="event-date">
         JULY 21-22, 2021 / ONLINE
       </h2>
     </section>
-    <section class="main-program">
+    <section class="main-program" id="program">
```
Comment (GzimAsani) [inline index.html:45]: - [x] Please adjust the colors and padding similar to the given design.

Yours:
Screenshot from 2021-11-16 17-57-38
Given design:
Screenshot from 2021-11-16 17-57-41

File: index.html [hunk 4]
```diff
@@ -64,14 +77,8 @@ <h4 class="lectures-title">CC Ignite</h4>
...
       <hr class="orange-line">
-      <article class="speaker-container">
-        <img class="speaker-img" src="./images/speakers/1.jpeg" alt="speaker img">
-        <div class="speaker-info">
-          <h4 class="speaker-name">Yochai Benkler</h4>
-          <p class="speaker-about">Product Designer</p>
-          <p class="talk-about">Create the career you're looking for</p>
-        </div>
-      </article>
+      <div class="grid-container">
```
Comment (GzimAsani) [inline index.html:80]: - [x] In this section the text in red should be `italic`

File: index.html (hunk 3 above)
Comment (GzimAsani) [reply index.html:45]: - [x] In the mobile version all of the `li` should be the same width and length
Screenshot from 2021-11-16 18-01-58

File: html/about.html [hunk 5]
```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
       <p class="creative-description">The logo of Design was decided through the logo competition</p>
```
Comment (GzimAsani) [inline html/about.html:40]: -  [x] The logo should have a displayed border similar to the given design

File: html/about.html [hunk 6]
```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
...
       <a class="logo" href="../index.html">Design & Content</a>
     </section>
-    <section class="past-events">
+    <section class="past-events" id="events">
```
Comment (GzimAsani) [inline html/about.html:45]: - [x] In this section for the desktop version the pictures should be horizontal:

Yours:
Screenshot from 2021-11-16 18-05-56
Original:
Screenshot from 2021-11-16 18-06-23

File: html/about.html [hunk 7]
```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
...
       <h5 class="past-title">See the past Design & Content Conferences </h5>
       <hr class="orange-line">
       <p class="past-description">Take a look at the last two Design & Content Conferences</p>
```
Comment (GzimAsani) [inline html/about.html:48]: - [x] This part of the footer should not be part of the about page for desktop version:
Screenshot from 2021-11-16 18-08-14

File: index.html [hunk 8]
```diff
@@ -6,16 +6,29 @@
...
+      <a class="user-bar-link" href="#">Sign in</a>
+    </div>
     <nav>
+      <a class="logo" href="#main">Design & Content</a>
       <img class="hamb-icon" src="./images/menu.png" alt="menu icon">
       <ul class="menu-links">
         <li class="menu-list close"><img class="hamb-close-btn" src="./images/close.png" alt="close button"></li>
-        <li class="menu-list"><a class="menu-link" href="./index.html">Home Page</a></li>
-        <li class="menu-list"><a class="menu-link" href="./html/about.html">About</a</li>
+        <li class="menu-list homepage"><a class="menu-link" href="./index.html">Home Page</a></li>
```
Comment (GzimAsani) [inline index.html:26]: - [x] In the mobile version there is no button to go back to the first page please add it in your hamburger

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @youmari                         
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] Please be reminded  for next projects that the changes should be asked to be merged from the feature bracn to the master branch
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (youmari) [reply README.md]: Actually, i did record a video I sent the link with a request review

File: index.html (hunk 2 above)
Comment (youmari) [reply index.html:34]: done!!

File: README.md (hunk 1 above)
Comment (youmari) [reply README.md]: done!!

File: index.html (hunk 4 above)
Comment (youmari) [reply index.html:80]: done this too !!

File: index.html (hunk 3 above)
Comment (youmari) [reply index.html:45]: this too !! is done

File: html/about.html (hunk 5 above)
Comment (youmari) [reply html/about.html:40]: i do it too  right now thanks

File: html/about.html (hunk 6 above)
Comment (youmari) [reply html/about.html:45]: done !!

File: index.html (hunk 8 above)
Comment (youmari) [reply index.html:26]: done it tooo

File: html/about.html (hunk 7 above)
Comment (youmari) [reply html/about.html:48]: i 've just changed it thanks again

**Comment by youmari** [comment]
I make all the required changes

**Comment by youmari** [comment]
# this the loom video describing the projects

https://www.loom.com/share/cbb5fa0b961f4493be61056686886635

**Review by D4nielJ — APPROVED** [review]
## Hello @youmari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

[OPTIONAL] I think you could add a link to the video on Loom in your README. It will be great documentation about the project and maybe a recruiter might see it in the future.

Also, props on the video! The quality is great and you explain professionally every feature developed in the project.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**